web-ts/src/components/FilterableCitySelector.tsx

```diff
@@ -1,28 +1,25 @@
-// @flow
```

Comment:
I do not really like the naming of these components. For me it is not clear, what file contains what. maybe we can rename them?
I also think somewhere in the filename there should be the keyword "List" as it is mostly a List?
Maybe you can have a look at this:
https://medium.com/@wittydeveloper/react-components-naming-convention-%EF%B8%8F-b50303551505
So maybe CitySelector should be LandingCityList or so? Not sure about the best solution, but the current one seems confusing - at least for me.

Reply:
Agreed, naming is not very good in quite some components. Also, many components are too big and legacy. Anyway, I think it is a very big task to adjust all naming and refactor the components as well, so imo it is out of the scope of this issue. I guess we also should get the web app up and running again relatively fast. Do you want to create an issue for this? Than you can choose a naming of your choice :)